Update VSCode install/debug instructions.

rkeithhill opened the pull request:
Fix #3257

<!--

If you are a PowerShell Team member, please make sure you choose the Reviewer(s) and Assignee for your PR.
If you are not from the PowerShell Team, you can leave the fields blank and the Maintainers will choose them for you. If you are familiar with the team, feel free to mention some Reviewers yourself.

For more information about the roles of Reviewer and Assignee, refer to CONTRIBUTING.md.

-->

rkeithhill committed:
Update VSCode install/debug instructions.

Fix #3257

rkeithhill committed:
Update to use semantic line - one sentence / line.

SteveL-MSFT commented on docs/learning-powershell/using-vscode.md:
```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
+You can press this button to add more PowerShell debug configurations. One handy configuration to add is **PowerShell: Launch Script**.
+With this configuration, you can specify a specific file with optional arguments that should be launched whenever you press F5 no matter which file is currently active in the editor.
+
+Once the debug configuration is established, you can select which configuration you want to use during a debug session by selecting one from the debug configuration dropdown in the **Debug** view's toolbar.
```
Should be 'drop-down'

SteveL-MSFT commented on docs/learning-powershell/using-vscode.md:
```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
+* Attach the debugger to a PowerShell host process
+
+Follow these steps to create your debug configuration file:
+1. Open the **Debug** view by pressing **Ctrl+Shift+D** (**Cmd+Shift+D** on Mac).
```
Add two spaces at the end of each line to get this formatted as a linebreak

rkeithhill replied:
I've made all the other changes. But I'm not sure I follow this one.  What is the issue?  I have VSCode set up to trim trailing whitespace which is why this probably changed.

SteveL-MSFT commented on docs/learning-powershell/using-vscode.md:
```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
-  Because some information regarding your scripts is needed for debugger to start executing your script, we need to set up the debug config first.
-  This is one-time process to debug PowerShell scripts under your current folder.
-  In our case, the "demo" folder.
+   When you do this, Visual Studio Code creates a directory and a file **.vscode\launch.json** in the root of your workspace folder.
```
Genereally, it seems file paths are in quotes, so if you put this in quotes, I think the spellchecker will be fine with it

SteveL-MSFT commented on docs/learning-powershell/using-vscode.md:
```diff
@@ -16,46 +16,46 @@ Editing with Visual Studio Code
-  *	**macOS**:         typing **code** in your terminal
+-	Launch the Visual Studio Code app by:
+    *	**Windows**:      typing **code** in your PowerShell session
+    *	**Linux**:        typing **code .** in your terminal
```
Perhaps Code required a path on Linux previously, but that doesn't seem the be the case anymore.  just `code` is fine for Linux

rkeithhill committed:
Make suggested changes.

SteveL-MSFT commented:
@rkeithhill looks like you'll need to add `vscode` as a valid word in `.spelling` for in the ` - docs/learning-powershell/using-vscode.md` section

rkeithhill committed:
Add vscode to .spelling file.

This term is required to specify the path to the launch.json file e.g.:  .vscode/launch.json.
